scripts/synonyms.json

```diff
@@ -0,0 +1,4 @@
+[
+     "postgresql, postgres, pg",
+     "kafka, kafak, kfaka"
```

Comment:
should "kafka, kafak, kfaka" better achieved with fuzzy matching?

Reply:
Yes, it absolutely should be :). I wrote this before the fuzzy matching PR went in, which makes this more useful for fundamental aliases rather than typos (like "pg").  I'll remove the kafka one, although I think it's good to seed the list with at least 2 entries so I'll find another one to add in its place.

Reply:
I've changed this for a different synonym example ("mirrormaker, mm2").